server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -167,9 +167,11 @@ func (w *Worker) Work(ctx workflow.Context) {
 			return
 		case process:
 			logger.Info(ctx, "Processing... ")
-			currentDeployment, err = w.deploy(ctx, w.latestDeployment)
 		case receive:
 			logger.Info(ctx, "Received unlock signal... ")
+			w.Scope.SubScopeWithTags(map[string]string{"signal_name": UnlockSignalName}).
```

Comment:
will these tags be used in other workflows too? If so, it'd be nice to make them consts in the metrics package and export them to avoid any typos. What do you think?

Reply:
It'll be used for other signals, so yeah I can probably make this a constant.